### PR TITLE
fix stringifyable identifier

### PR DESCRIPTION
**Pull request by soyuka**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #924 
| License       | MIT
| Doc PR        | na

### CODE REVIEW

File: src/Bridge/Symfony/Routing/IriConverter.php [hunk 1]
```diff
@@ -126,6 +126,9 @@ private function getIdentifiersFromItem($item): array
 
             if (!is_object($identifiers[$propertyName])) {
                 continue;
+            } else if (method_exists($identifiers[$propertyName], '__toString')) {
```
Comment (eps90) [inline src/Bridge/Symfony/Routing/IriConverter.php]: If projects keeps PSR-2 rules, there should be `elseif` instead of `else if`

**Comment by teohhanhui** [comment]
We might want to refactor the code a bit, since we keep using
```php
$identifiers[$propertyName]
```
lol...

File: src/Bridge/Symfony/Routing/IriConverter.php (hunk 1 above)
Comment (dunglas) [inline src/Bridge/Symfony/Routing/IriConverter.php]: `elseif`

**Comment by soyuka** [comment]
Refactored, fixed, and added a test.

**Comment by dunglas** [comment]
Thanks @soyuka

**Comment by dkarlovi** [comment]
@soyuka awesome, thanks a lot! \o/

**Comment by teohhanhui** [comment]
@soyuka The test is incorrect. The entity's `getId` method should return an object which implements `__toString`.

**Comment by soyuka** [comment]
Hmm, wasn't sure about this one. Should have left a note. I'm opening a new pr to patch this.

**Pull request merged**